### PR TITLE
Add debug script to react-server-test-pages

### PR DESCRIPTION
**Pull request by lidawang**
Let us use `npm run debug` in `react-server-test-pages`! I didn't know whether or not to pin a version on `node-inspector`, please let me know if I should and which one to pin to!

### CODE REVIEW

File: packages/react-server-test-pages/package.json [hunk 1]
```diff
@@ -19,5 +20,8 @@
     "react-server-cli": "^0.2.6",
     "react-server-data-bundle-cache": "^0.2.6",
     "superagent": "1.2.0"
+  },
+  "devDependencies": {
+    "node-inspector": ""
```
Comment (gigabo) [inline packages/react-server-test-pages/package.json]: No version?

**Comment by gigabo** [comment]
This is awesome @lidawang!  Worth mentioning in `packages/react-server-test-pages/README.md`.

File: packages/react-server-test-pages/package.json (hunk 1 above)
Comment (lidawang) [inline packages/react-server-test-pages/package.json]: Yea tbh I'm not sure which to pick! The latest stable version that I used for getting this to work is 0.12.8 - I wasn't sure if I should pin to that specific version or accept patches (~0.12.8?), etc. Any guidance?

Comment (gigabo) [inline packages/react-server-test-pages/package.json]: Generally the easiest way to add a dependency like this is to use:

```bash
npm install --save-dev node-inspector
```

That will install it and add it to the `devDependencies` in package.json for you, with an appropriate version.

File: packages/react-server-test-pages/README.md [hunk 2]
```diff
@@ -19,4 +19,6 @@ Then hit http://localhost:3000/ to see what's available.
 If you make changes in `packages/react-server` you'll need to `CTRL-C` and
 re-run the last line of the setup to pull them in.
 
+If you wish to run the test server in debug mode, simply replace `npm start` with `npm run debug`.
+
```
Comment (gigabo) [inline packages/react-server-test-pages/README.md]: :metal:

**Pull request merged**